config/jobs/kubernetes/kops/kops-presubmits-e2e.yaml

```diff
@@ -1142,6 +1142,71 @@ presubmits:
       testgrid-days-of-results: '90'
       testgrid-tab-name: pull-kops-e2e-aws-dns-none
 
+# {"cloud": "aws", "container_runtime": "containerd", "distro": "u2204", "extra_flags": "--api-loadbalancer-type=public --discovery-store=s3://k8s-kops-prow/discovery", "k8s_version": "stable", "kops_channel": "alpha", "networking": "calico"}
+  - name: pull-kops-e2e-aws-lb
```

Comment:
How does this end "-lb" when build_jobs.py has the name end with "-nlb"?

When I run build_jobs.py, I get uncommitted changes.

Reply:
Fixed in https://github.com/kubernetes/test-infra/pull/27839.